python/pyspark/sql/streaming/query.py

```diff
@@ -188,7 +192,7 @@ def awaitTermination(self, timeout: Optional[int] = None) -> Optional[bool]:
 
         Return whether the query has terminated or not within 5 seconds
 
-        >>> sq.awaitTermination(5)
+        >>> sq.awaitTermination(5) # doctest: +SKIP
```

Comment:
Why are these needed? This won't be tested in connect yet, right?

Reply:
Right.. This actually won't be tested in both connect and non-connect I believe (@HyukjinKwon please correct me if I'm wrong) , just some temporary pain we have to bear with

Reply:
What is the pain? What fails if you remove this diff?

Reply:
Lets not skip any tests. Any skip should have a TODO comment.

Reply:
Sure! I've moved the TODO to the top of these methods to avoid them showing up in the PySpark doc

Reply:
If we're absolutely sure that we will enable this back very soon, I am fine. As @rangadi said, I tried my best to avoid skipping the tests, but I did few times when I am about to enable it back very soon.

Reply:
Thanks! Done